packages/kit/src/runtime/app/env.js

```diff
@@ -1,28 +1,11 @@
-/* global __SVELTEKIT_BUILD__ */
-
 /**
  * @type {import('$app/env').browser}
  */
 export const browser = !import.meta.env.SSR;
 
-/**
- * @type {import('$app/env').server}
- */
-export const server = !!import.meta.env.SSR;
-
 /**
  * @type {import('$app/env').dev}
```

Comment:
```suggestion
 * Whether the dev server is running.
 * This is not guaranteed to correspond to `process.env.NODE_ENV` or `import.meta.env.MODE`.
 * @type {import('$app/env').dev}
```

Reply:
I think this probably needs to go here instead: https://github.com/sveltejs/kit/blob/ad09e6cb012165475f12e672b74bfad28d7100c9/packages/kit/types/ambient.d.ts#L71-L80

Reply:
yeah. github won't let me comment that line since it's not changed, so I'll let you push it if you're happy with it

Reply:
done